org.polymc.PolyMC.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
```

Comment:
Ideally, this should be maintained upstream. Not a blocker, though.

Reply:
upstream does have it but some things (like icon) need to be different here

Reply:
It can be modified with desktop-file-edit.